Sources/NIOCore/AsyncSequences/NIOAsyncSequenceProducer.swift

```diff
@@ -127,13 +129,13 @@ public struct NIOAsyncSequenceProducer<
     /// caller. This is due to the fact that deiniting the sequence is used as part of a trigger to terminate the underlying source.
     ///
     /// - Parameters:
-    ///   - element: The element type of the sequence.
+    ///   - elementType: The element type of the sequence.
     ///   - backPressureStrategy: The back-pressure strategy of the sequence.
     ///   - delegate: The delegate of the sequence
     /// - Returns: A ``NIOAsyncSequenceProducer/Source`` and a ``NIOAsyncSequenceProducer``.
     @inlinable
     public static func makeSequence(
-        of elementType: Element.Type = Element.self,
+        elementType: Element.Type = Element.self,
```

Comment:
This is breaking the public API; however, we haven't shipped a release with this yet so I would like to align the naming before we do that. cc @Lukasa @glbrntt